### PR TITLE
doc: translate hooks-intro

### PR DESCRIPTION
**Pull request by chentsulin**
Ready for first time review.

### CODE REVIEW

File: content/docs/hooks-intro.md [hunk 1]
```diff
@@ -1,110 +1,110 @@
...
     </div>
   );
 }
 ```
 
-This new function `useState` is the first "Hook" we'll learn about, but this example is just a teaser. Don't worry if it doesn't make sense yet!
+這個新 function `useState` 將會是我們學習的第一個「Hook」，不過這只是一個初窺的範例。如果這現在看起來並不合理，請別擔心！
 
-**You can start learning Hooks [on the next page](/docs/hooks-overview.html).** On this page, we'll continue by explaining why we're adding Hooks to React and how they can help you write great applications.
+**你可以開始[在下一頁](/docs/hooks-overview.html)學習 Hook。**在這一頁，我們將會繼續解釋為什麼要把 Hook 添加到 React 以及他們如何幫助你寫出好的應用程式。
```
Comment (neighborhood999) [inline content/docs/hooks-intro.md]:
```suggestion
**你可以開始[在下一頁](/docs/hooks-overview.html)學習 Hook。**在這一頁，我們將會繼續解釋為什麼要把 Hook 增加到 React 以及他們如何幫助你寫出好的應用程式。
```

**Comment by BruceWeng** [comment]
Hi @chentsulin, 

I also finished translating Introducing Hooks just now in doc: Translate Hooks #32
. How about we distribute ownership in the following topic about Hooks. 

I am actually open to translate any of them, let me know if you have a specific one you would love to do. Thanks!

@chloewlin can label ownership on Translation Progress Page as well.

File: content/docs/hooks-intro.md [hunk 2]
```diff
@@ -1,110 +1,110 @@
...
 
 function Example() {
-  // Declare a new state variable, which we'll call "count"
+  // 宣告一個新的 state 變數，並稱它為「count」
   const [count, setCount] = useState(0);
 
   return (
     <div>
-      <p>You clicked {count} times</p>
+      <p>你點擊了 {count} 次</p>
```
Comment (chloewlin) [inline content/docs/hooks-intro.md]: We agreed not to translate code block. Can you revert this back to English? Thanks!

File: content/docs/hooks-intro.md [hunk 3]
```diff
@@ -1,110 +1,110 @@
...
       <button onClick={() => setCount(count + 1)}>
-        Click me
+        點擊我
```
Comment (chloewlin) [inline content/docs/hooks-intro.md]: Revert to English.

File: content/docs/hooks-intro.md (hunk 1 above)
Comment (chentsulin) [reply content/docs/hooks-intro.md]: How about `把 Hook 加到 React`

**Comment by chentsulin** [comment]
Hi @BruceWeng
Sure, I'd like to take `hooks-effect.md` and maybe take next one after I finish it. Feel free to take another one you like.

File: content/docs/hooks-intro.md (hunk 1 above)
Comment (neighborhood999) [reply content/docs/hooks-intro.md]: Agree! 👌

**Comment by BruceWeng** [comment]
@chentsulin
No problem. I will take `hooks-overview.md`. Once I've finished, I will do `hooks-state.md`.

**Pull request merged**